docs/materials/day2/part1-ex1-submit-refresher.md

```diff
@@ -14,68 +14,30 @@ Where in the world are my jobs?
-It is not important to understand how the internals of the script works.
-When called without any arguments, the script will return the latitude and longitude coordinates of the machine from
-which it is run.
-
-```
-#!/bin/env python
-
-import sys
-import socket
-import urllib2
-import json
-import time
-
-hostnames = set()
-if len(sys.argv) == 1:
-    hostnames.add(socket.getfqdn())
-else:
-    for filename in sys.argv[1:]:
-        try:
-            with open(filename, 'r') as f:
-                hostnames.update(set([x.strip() for x in f.readlines()]))
-        except IOError:
-            pass
-
-hostnames.discard('')
-for host in hostnames:
-    try:
-        ipaddr = socket.gethostbyname(host)
-        for i in range(1,4):
-            try:
-                response = urllib2.urlopen('http://www.freegeoip.net/json/' + ipaddr).read()
-                json_response = json.loads(response)
-                lat = json_response['latitude']
-                lon = json_response['longitude']
-
-                if int(lat) is 0 and int(lon) is 0:
-                    exit("www.freegeoip.net could not find associated lat/lon coordinates.")
-                print "%s,%s" % (lat, lon)
-                break
-            except urllib2.HTTPError:
-                time.sleep(3**i)
-                pass
-    except socket.gaierror:
-        pass
-```
-
 ### Geolocating several machines
 
-Now, let’s try to use this Python script and remember some basic HTCondor ideas from yesterday!
+Now, let’s try to remember some basic HTCondor ideas from yesterday!
 
 1.  Log in to `learn.chtc.wisc.edu`
 1.  Create and change into a new folder for this exercise, for example `tuesday-1.1`
-1.  Save the Python script above as a file named `location.py`
-1.  As always, ensure that your script has the proper permissions (hint: try running it from the command line)
-1.  Create a submit file that generates **ten** jobs that run `location.py` and uses the `$(Process)` macro to write
-    different `output` and `error` files.
+1.  Download the geolocation code:
+
+        :::console
+        username@learn $ wget http://proxy.chtc.wisc.edu/SQUID/osgschool18/location-wrapper.sh
+                         http://proxy.chtc.wisc.edu/SQUID/osgschool18/wn-geoip.tar.gz
+
+    You will be using `location-wrapper.sh` as your executable and `wn-geoip.tar.gz` as an input file.
+
+1.  As always, ensure that your executable has the proper permissions (hint: try running it from the command line)
+1.  Create a submit file that generates **one hundred** jobs that run `location-wrapper.sh`, transfers `wn-geoip.tar.gz`
+    as an input file, and uses the `$(Process)` macro to write different `output` and `error` files with the following
+    requirements:
+
+        Requirements = (HAS_CVMFS_oasis_opensciencegrid_org =?= TRUE)
+
     Try to do this step without looking at materials from yesterday.
     But if you are stuck, see [yesterday’s exercise 2.4](/materials/day1/part2-ex2-queue-n.md).
```

Comment:
Need's to say "yesterday's exercise 2.4" to match the filename (link is correct).